.mailmap

```diff
@@ -7,3 +7,96 @@
 <ind@henn.et> <shennetsind@54d463be-8e91-2dee-dedb-b68131a5f0ec>
 <Kenpachi.Developer@gmx.de> <kenpachi2k11@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<umage@netvor.sk> <ultramage@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<ai4rei@users.noreply.github.com> <ai4rei@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<Kenji.Ito@gmx.de> <Playtester@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<masao@gmx.ch> <masao87@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<piotr.halaczkiewicz@gmail.com> <gepard1984@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<piotr.halaczkiewicz@gmail.com> <Gepard@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<missxantara@gmail.com> <xantara@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<lightaisme@gmail.com> <glighta@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<brianluau@users.noreply.github.com> <brianluau@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<harrison@isabarwon.com> <calciumkid@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<fw@f-ws.de> <blacksirius@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<jmish1987@gmail.com> <jmanfffreak@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<paradox924x@gmail.com> <Paradox924x@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<paradox924x@gmail.com> <Paradox924X@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<yomanda@gmail.com> <Yommy@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<masterofmuppets89@gmail.com> <MasterOfMuppets@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<flaviojs2005@gmail.com> <flaviojs@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<flaviojs2005@gmail.com> <FlavioJS@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+<flaviojs2005@gmail.com> <flaviojs2005@54d463be-8e91-2dee-dedb-b68131a5f0ec>
+
+# old aliases
+
+<haru@dotalux.com> <haru@originsro.org>
+<ind@henn.et> <int@henn.et>
+<ind@henn.et> <notind@gmail.com>
+<ind@henn.et> <shennetsind@users.noreply.github.com>
+<dastgirp@gmail.com> <dastgir@users.noreply.github.com>
+<dastgirp@gmail.com> <dastgirpojee@rocketmail.com>
+<michieru-kun@hotmail.com> <Michieru@users.noreply.github.com>
+<michieru-kun@hotmail.com> <michieru@0-mail.com>
+<jedzkie13@rocketmail.com> <Jedzkie@users.noreply.github.com>
+<jedzkie13@rocketmail.com> <eikzdej@gmail.com>
+<jedzkie13@rocketmail.com> <jedzkie999@gmail.com>
+<Equinox1991@gmail.com> <Emistry@users.noreply.github.com>
+<gauvain.dauchy@free.fr> <alige@users.noreply.github.com>
+<guilherme.menaldo@outlook.com> <guilherme-gm@users.noreply.github.com>
+<dev@herc.ws> <Hercules@efficiently.awesome>
+<streusel@gravity.co.kr> <advance_me@hotmail.de>
+<streusel@gravity.co.kr> <m.me1@live.de>
+<ibrahem.h.basyone@gmail.com> <hemagx2@gmail.com>
+<Asheraf@users.noreply.github.com> <acheraf1998@gmail.com>
+<sagunkho@hotmail.com> <sagunxp@gmail.com>
+
+
+
+
+
+# missing from github:
+# - skotlex
+# - L0ne_W0lf
+# - Lupus
+# - celest
+# - Lance
+# - zephyrus
+# - amber
+# - Inkfish
+# - toms
+# - shadowlady
+# - codemaster
+# - rud0lp20
+# - Vicious
+# - samuray22
+# - Kevin
+# - eathenabot
+# - shadow
+# - mc_cameri
+# - evera
+# - SinSloth
+# - epoque11
+# - momacabu
```

Comment:
This sounds like @macabu, but i could be wrong

Reply:
Hello! Yes, that's me (momacabu) :smile: